### PR TITLE
Okta SCIM page updates

### PR DESCRIPTION
**Pull request by yoejigu**
### Proposed changes

- Made the notice to reach out to Okta to enable SCIM option if it is missing more prominent 

- Split up Enabling group provisioning and configuring group provisioning into two sections

- Changed the link names to match the new section name

- Made the warning to ensure that all users are assigned the app in Okta before configuring Push groups more prominent 

### Related issues (optional)

https://github.com/pulumi/docs/issues/11619

### CODE REVIEW

File: theme/yarn.lock [hunk 1]
```diff
@@ -197,6 +197,13 @@
   dependencies:
     purgecss "^4.0.3"
 
+"@growthbook/growthbook@^1.2.1":
```
Comment (thoward) [inline theme/yarn.lock:200]: In theory, your above markdown changes should not require a update to `
yarn.lock`. Any idea why this is here? Can we remove changes from the commit? (e.g. if we need to bump library versions, that should be done in a separate PR and investigated with depth separately from a content update)

**Review by thoward — APPROVED** [review]
Thanks so much, Yosef! This all looks great.

Can you describe how you tested this? Is it possible to record a short screen capture of you doing these steps, and showing that they work? These kinds of docs are hard to verify because we can't e.g. use a code testing framework to verify that this works as described.

File: theme/yarn.lock (hunk 1 above)
Comment (yoejigu) [reply theme/yarn.lock:200]: Hey Troy, that change in the yarn.lock showed up after running the "make ensure" command so I remove that change if it shouldn't be included

**Comment by yoejigu** [comment]
https://github.com/user-attachments/assets/93af15c6-e623-4642-8e9c-c63986a85214
